pkg-manager/core/src/install/allProjectsAreUpToDate.ts

```diff
@@ -20,13 +20,17 @@ export async function allProjectsAreUpToDate (
   projects: Array<ProjectOptions & { id: string }>,
   opts: {
     autoInstallPeers: boolean
+    excludeLinksFromLockfile: boolean
```

Comment:
The test cases in [allProjectsAreUpToDate](https://github.com/pnpm/pnpm/blob/main/pkg-manager/core/test/allProjectsAreUpToDate.test.ts) needed to be updated since param `excludeLinksFromLockfile` is required